src/pages/Environments.js

```diff
@@ -141,7 +141,7 @@ const Environments = () => {
   return h(FooterWrapper, [
     h(TopBar, { title: 'Cloud Environments' }),
     div({ role: 'main', style: { padding: '1rem', flexGrow: 1 } }, [
-      div({ style: { ...Style.elements.sectionHeader, textTransform: 'uppercase', marginBottom: '1rem' } }, ['Your cloud environments']),
+      h2({ style: { ...Style.elements.sectionHeader, textTransform: 'uppercase', marginBottom: '1rem' } }, ['Your cloud environments']),
```

Comment:
This change seems to have added some margin to the top, was that intentional? (I think it may just need an explicit margin set for marginTop to avoid inheriting from the header.)

Reply:
Right, true, I see that the heading style adds a bit of top and bottom padding which the original styling did not have.

If preserving the original spacing on this page is important, I'd be happy to 0 it out or go back to using a `div` and add `role="header"`. I would argue, however, that headings serve a visual purpose, and it's generally appropriate to offset them from the content a little bit.

Reply:
Looks like the margins were being set explicitly before anyway. Just pushed an update to zero out any padding, as well at the top margin for the first h2 (the second h2 still has margins both top and bottom)

Reply:
Thank you! I am not super worried about it (and even prefer a little more padding), but if we don't need it for accessibility, I would rather not make display changes without confirming with the UX team.

Reply:
Sure, makes senes to me :)